components/DetailsPageHighlights/detailspagehighlights.scss

```diff
@@ -0,0 +1,183 @@
+.bx--artifact-details-highlight-container {
+
+  & .bx--createArtifactFlexGroupColumn {
+    margin-bottom: rem(35px);
+  }
+
+  & .bx--createArtifactFlexGroupColumn {
+    display: flex;
+    flex-direction: column;
+
+    &.bx--hide {
+      display: none;
+    }
+  }
+
+  & .bx--title-label {
+    font-size: 1.55rem;
+    font-weight: 200;
```

Comment:
Font-weight: 200 is a deprecated weight, swap to `300` for light and  `400` for regular